File: Backend/AppointmentsAPI/Domain/HealthInsurancePlans.cs
namespace AppointmentsAPI.Domain;

public static class HealthInsurancePlans
{
    public static readonly IReadOnlyList<string> All = new[]
    {
        "Unimed", "Amil", "SulAmérica", "Bradesco Saúde",
        "NotreDame Intermédica", "Hapvida", "São Francisco",
        "Porto Seguro Saúde", "Prevent Senior", "Golden Cross"
    };
}

File: Frontend/src/lib/health-insurance-plans.ts
export const HEALTH_INSURANCE_PLANS = [
  "Unimed",
  "Amil",
  "SulAmérica",
  "Bradesco Saúde",
  "NotreDame Intermédica",
  "Hapvida",
  "São Francisco",
  "Porto Seguro Saúde",
  "Prevent Senior",
  "Golden Cross",
] as const;

export type HealthInsurancePlan = (typeof HEALTH_INSURANCE_PLANS)[number];
